app/models/save_return.rb

```diff
@@ -0,0 +1,9 @@
+class SaveReturn < ApplicationRecord
+  after_initialize :set_default_values
+
+  private
+
+  def set_default_values
+    self.expires_at ||= Time.now + 28.days
```

Comment:
this is a placeholder and will change once we agree on how client will hand this information over

Reply:
👍